src/styles.scss

```diff
@@ -227,3 +229,8 @@ app-root {
   width: min(100vw, 1920px);
   margin-inline: auto;
 }
+
+app-page-not-found {
```

Comment:
możemy dodawać style do komponentu wykorzystując psuedoselektor `:host` specyficzny dla angulara. odnosi się wtedy ten selektor do danego komponentu gdzie dodaliśmy style. dzieki temu nie brudzimy globalnego arkusza styli

Reply:
style przeniesione do pliku page-not-found.scss 
![image](https://user-images.githubusercontent.com/44174415/211826363-cd870044-778d-4ef7-915e-f779149d23a0.png)